java/src/main/java/gov/doc/ntia/sigmf/ext/annotation/sensor/SensorAnnotation.java

```diff
@@ -20,14 +14,6 @@ public class SensorAnnotation extends Annotation {
   @JsonProperty(value = "ntia-sensor:overload", required = false)
   protected Boolean overload;
 
-  @JsonProperty(value = "ntia-sensor:attenuation_setting_sigan", required = false)
```

Comment:
should these not be deleted if this class is deprecated?

Reply:
Restored.